hack/verify-gofmt.sh

```diff
@@ -52,3 +52,14 @@ if [[ -n "${bad_files}" ]]; then
   echo "${bad_files}"
   exit 1
 fi
+
+# Find comments not prefaced with a space.  gofmt doesn't do this.
+# Multiple grep statements is easier to extend debug, although not elegant.
+bad_files=$(find_files | xargs grep '//[^ ]' | grep -v "://" | grep -v swagger | grep -v "\"")
```

Comment:
Or my own cmd/libs/go2idl.

But really I kinda think we don't urgently need this check, I don't usually
see it wrong in PRs.

On Wed, Nov 18, 2015 at 1:52 PM, Eric Paris <notifications@github.com>
wrote:

> In hack/verify-gofmt.sh
> <https://github.com/kubernetes/kubernetes/pull/17454#discussion_r45266137>
> :
>
> > @@ -52,3 +52,14 @@ if [[ -n "${bad_files}" ]]; then
> >    echo "${bad_files}"
> >    exit 1
> >  fi
> > +
> > +# Find comments not prefaced with a space.  gofmt doesn't do this.
> > +# Multiple grep statements is easier to extend debug, although not elegant.
> > +bad_files=$(find_files | xargs grep '//[^ ]' | grep -v "://" | grep -v swagger | grep -v "\"")
>
> careful, he's likely to tell you to use https://golang.org/pkg/go/parser/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17454/files#r45266137>.
>